app/src/main/kotlin/com/wire/android/media/CallRinger.kt

```diff
@@ -49,11 +49,16 @@ class CallRinger @Inject constructor(private val context: Context) {
         mediaPlayer?.isLooping = isLooping
     }
 
-    fun ring(resource: Int, isLooping: Boolean = true) {
+    fun ring(
+        resource: Int,
+        isLooping: Boolean = true,
+        isIncomingCall: Boolean = true
+    ) {
         stop()
-        vibrateIfNeeded()
+        if (isIncomingCall)
+            vibrateIfNeeded()
```

Comment:
Maybe the `isIncomingCall` value could also be passed down to `vibrateIfNeeded` so the logic will be done there in case in the future we want to use from somewhere else and wouldn't need to be doing the verification before calling `vibrateIfNeeded`